tools/Unfolding.py

```diff
@@ -41,13 +41,11 @@ def __init__( self,
         self.fakes = fakes
         self.response = response
         self.data = None
-        self.unfolded_closure = None
         self.unfolded_data = None
         self.unfoldObject = None
         self.unfoldResponse = None
-        self.closure_test = None
         self.verbose = verbose
-        self.tau = tau
+        self.tau = float(tau)
```

Comment:
there was a bug where I put in 19 and forgot the dot.
tau can have both integer and float values (a real number), but it needs always to be of type float in order to call the correct constructor